enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/APIFactory.java

```diff
@@ -64,10 +66,18 @@ public void addApis(List<Api> apis) {
         ConcurrentHashMap<String, API> newApis = new ConcurrentHashMap<>();
 
         for (Api api : apis) {
-            RestAPI enforcerApi = new RestAPI();
-            enforcerApi.init(api);
-            String apiKey = getApiKey(enforcerApi);
-            newApis.put(apiKey, enforcerApi);
+            if (api.getApiType().equals(APIConstants.ApiType.REST)) {
```

Comment:
This should be the other way.
```suggestion
            if ((APIConstants.ApiType).REST.equals(api.getApiType())) {
```